profiles/OleanjiKingCode.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/75235148/193233146-93bb266c-9ad7-4021-970b-96cb303501b1.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/193316450-19e6aa4f-a0a6-44b5-8e2e-fec79dc581e6.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/193316450-19e6aa4f-a0a6-44b5-8e2e-fec79dc581e6.png",
```